bin/population/pycbc_multiifo_pastro

```diff
@@ -0,0 +1,396 @@
+    plt.savefig(args.plot_dir + 'fg_stat_vs_bgpdf.png')
+    plt.close()
+
+    for cty in col:
+        in_type = fg.ctype == cty
+        plt.plot(fg.stat[in_type], allfgbg[in_type], col[cty]+'+',
+                 ms=3, alpha=0.6, label=cty)
+    plt.grid(True)
+    plt.legend()
+    plt.xlabel('Rank statistic')
+    plt.ylabel('fg/bg Bayes factor')
+    plt.xlim(xmax=args.plot_max_stat)  # zoom to see not-super-loud events
+    plt.savefig(args.plot_dir + 'fg_stat_vs_fgbg_bf.png')
+    plt.semilogx()
+    plt.xlim(args.stat_threshold, 1.1 * fg.stat.max())
+    plt.savefig(args.plot_dir + 'fg_logstat_vs_fgbg_bf.png')
+    plt.close()
+
+    for cty in col:
+        in_type = fg.ctype == cty
+        plt.loglog(fg.stat[in_type], fg.ifar[in_type], col[cty]+'+',
+                   ms=3, alpha=0.6, label=cty)
+    plt.grid(True)
+    plt.legend()
+    plt.xlabel('Rank statistic')
+    plt.ylabel('IFAR (yr)')
+    plt.savefig(args.plot_dir + 'fg_stat_vs_ifar.png')
+    plt.close()
+
+# add in prior event info
+if args.prior_event_info:
+    logging.warn('Adding events from', args.prior_event_info)
+    prior_info = np.genfromtxt(args.prior_event_info, names=True)
+    allfgbg = np.append(allfgbg, prior_info['ln_bayes_factor'])
+else:  # no prior events!
+    prior_info = {'stat': np.array([]),
+                  'ifar': np.array([]),
+                  'gps': np.array([]),
+                  'bayes_factor': np.array([])}
+
+# do the calculation
+foreground_count_dist = \
+      fgmcl.count_posterior(allfgbg, laguerre_n=args.laguerre_degree,
+                            Lambda0=bg.norm, name='foreground count posterior',
+                            prior=args.power_law_prior)
+
+# Roulet et al. argue that information on rate scales with sum (p_astro^2)
+logging.info('sum of p_astro^2',
+             ((1 - foreground_count_dist.p_bg(allfgbg)) ** 2.).sum())
+
+fgmc_plots.odds_summary(
+    args,
+    np.append(fg.stat, prior_info['stat']),
+    np.append(fg.ifar, prior_info['ifar']),
+    foreground_count_dist.p_bg(allfgbg),
+    args.ntop,
+    times=np.append(fg.gpst, prior_info['gps']),
+    name='foreground events',
+    plot_extensions=['.png'],
+)
+
+args.plot_limits = None
+cmed, cminus, cplus = fgmc_plots.dist_summary(
+    args,
+    foreground_count_dist,
+    plot_extensions=['.png'],
+    middle='median',
+    credible_intervals=[0.90],
+)
+
+# if VT and error are provided, convert signal PDF to rate PDF
+if args.vt_mean:
+    if args.vt_err is None:
+        logging.warn('NB : no VT error provided, will use 0!')
+        args.vt_err = 0.
+    logging.info('Rate in units of ' + args.rate_units + ' ...')
+    print('Rate without VT uncertainty :')
+    print('%.1f_{-%.1f}^{+%.1f}' % (
+        cmed / args.vt_mean, abs(cminus) / args.vt_mean, cplus / args.vt_mean))
+    import rate_functions
+    fracerr = args.vt_err / args.vt_mean
+    # attr that can arbitrarily rescale counts in rate calculation : don't do this
```

Comment:
"don't do this"?

Reply:
Don't rescale by having a 'scale' different from 1 ...